File: trail1/장학금/scholarship.py
lst = list(map(int, input().split()))
A, B = lst[0], lst[1]
score = 0
if A >= 90:
    if B >= 95:
        print(100000)
    elif B >= 90:
        print(50000)
    else:
        print(0)
else:
    print(0)

File: trail1/장학금/scholarship.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        var(a, b) = (int.Parse(arr[0]), int.Parse(arr[1]));
        int score = 0;
        if (a >= 90 && b >= 95) {score = 100000;}
        else if (a >= 90 && b >= 90) {score = 50000;}
        Console.WriteLine(score);
    }
}
